source/fab/steps/analyse.py

```diff
@@ -132,8 +129,7 @@ def analyse(
 
     # todo: these seem more like functions
     fortran_analyser = FortranAnalyser(config=config,
-                                       std=std,
-                                       ignore_dependencies=ignore_dependencies)
+                                       std=std)
```

Comment:
Since the `ignore_dependencies` argument was restored the use of it should probably be restored too.